data/zones.lua

```diff
@@ -160,19 +191,19 @@ local function createBlips()
             if v.type == 'barber' then
                 spriteId = 71
                 blipColor = 0
-                blipname = 'Barber'
+                blipname = "Barber Shop"
             elseif v.type == 'clothing' then
                 spriteId = 73
                 blipColor = 0
-                blipname = 'Clothing Store'
+                blipname = "Clothing Store"
             elseif v.type == 'tattoos' then
                 spriteId = 75
                 blipColor = 4
-                blipname = 'Tattoo Parlor'
+                blipname = "Tattoo Shop"
```

Comment:
revert the name u cant buy the tattoo u are applying on ur body so it should be parlor and same for surgeon and use single quotes